powershell/VstsTaskSdk/LoggingCommandFunctions.ps1

```diff
@@ -201,14 +201,17 @@ function Write-SetResult {
         [Parameter(Mandatory = $true)]
         [string]$Result,
         [string]$Message,
+        [Parameter()]
+        [bool]$Done,
```

Comment:
use `[switch]` (true to the powershell way) instead of `[bool]`

Reply:
I'll implement this change!

Reply:
thanks. otherwise this lgtm.

Make sure the E2E scenario works before you merge.

Then after you merge this, sync w/ danny to release the new lib version, and consume the new version in the command line task and powershell task.